### PR TITLE
Add 'default': True to the eam metadata

### PR DESCRIPTION
**Pull request by aaronayres35**
This PR simply sets default to true in the eam metadata so that one can run the application from the command line through edm

### CODE REVIEW

**Review by rahulporuri — APPROVED** [review]
LGTM

**Comment by aaronayres35** [comment]
CI is failing with:
```
[EXECUTING] edm run -e etsdemo-test-3.6-pyside2 -- python -m unittest discover -v etsdemo
Shell 'sh' is not supported, things may not work properly (supported shells are bash, zsh)
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-runner'
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-runner'
Could not initialize GLX
Command '['edm', 'run', '-e', 'etsdemo-test-3.6-pyside2', '--', 'python', '-m', 'unittest', 'discover', '-v', 'etsdemo']' returned non-zero exit status 250.
/bin/bash /home/runner/work/_actions/GabrielBB/xvfb-action/v1/cleanup.sh
No xvfb processes to kill
Error: The process '/usr/bin/xvfb-run' failed with exit code 1
```

**Comment by rahulporuri** [comment]
Can you open an issue for the failing pyside2 job? Or maybve roll back pyside2 to an older version or use the version from edm. I'm not seeing any failures on windows with pyside2 so if we aren't able to make the CI pass, let's just go ahead and merge this.

**Comment by aaronayres35** [comment]
I've opened #1572 

I am unsure why pyside2 had been pinned to 5.11 previously.  Should we unpin it here (it is unpinned in traitsui itself)

File: ets-demo/etstool.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def install(runtime, toolkit, environment, editable):
     # pip install pyside2, because we don't have it in EDM yet
     if toolkit == 'pyside2':
         commands.append(
-            "edm run -e {environment} -- pip install pyside2==5.11"
```
Comment (aaronayres35) [inline ets-demo/etstool.py:197]: ref: #1452, but #1448 is fixed so I believe this is okay

**Review by rahulporuri — APPROVED** [review]
Still LGTM

**Pull request merged**